### DISCUSSION TITLE
reCAPTCHA v3 avec Gravity Forms Wordpress

### DISCUSSION THREAD

**Discussion by supportzandkologan** [AmauriC/tarteaucitron.js#1272] [Q&A]
Bonjour,

Avez-vous une solution pour intégrer reCAPTCHA v3 dans Tarteaucitron lorsque reCAPTCHA est chargé via Gravity Forms sur Wordpress ?

**Comment by AmauriC** [comment]
Bonjour,
De mémoire non, recaptcha dans les formulaires WordPress ne fonctionne généralement pas bien avec tarteaucitron :/
